samples/app-complete-auth/nodejs/README.md

```diff
@@ -119,6 +133,14 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 ### 2. Setup NGROK
 1) Run ngrok - point to port 3978
 
+    ```bash
+   ngrok http 3978 --host-header="localhost:3978"
+   ```
+   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
+
+   ```bash
+   devtunnel host -p 3333 --allow-anonymous
```

Comment:
Why dev tunnel using port ? 3333

ngRok you mentioned 3978

Reply:
duplicated, Fixed this comment